ff/serving/metadata/job.go

```diff
@@ -0,0 +1,101 @@
+package metadata
+
+import (
+	"fmt"
+)
+
+type JobType string
+
+const (
+	EXAMPLE_JOB JobType = "Example Job"
+)
+
+type EtcdStorage struct {
+	ResourceType ResourceType //Resource Type. For use when getting stored keys
+	StorageType  StorageType  //Type of storage. Resource or Job
+	Message      []byte       //Contents to be stored
+	JobType      JobType
+}
+
+func (config EtcdConfig) ParseJob(res EtcdStorage, jobType Job) (Job, error) {
+	if res.StorageType != JOB {
+		return nil, fmt.Errorf("payload is not job type")
+	}
+	if err := json.Unmarshal(res.Message, jobType); err != nil {
+		return nil, err
+	}
+	return jobType
+	
+}
+
+func (lookup etcdResourceLookup) findJobType(t JobType) (Job, error) {
```

Comment:
Might want to change etcdResourceLookup to a different type. This currently implements the ResourceLookup interface which probably isnt necessary for job management. 

Could create something like 
`type etcdJobLookup struct {
	connection EtcdConfig
}`

which would function the same way but doesnt implement ResourceLookup